.devcontainer/Dockerfile

```diff
@@ -0,0 +1,5 @@
+ARG VARIANT="3.10-bullseye"
+FROM mcr.microsoft.com/vscode/devcontainers/python:0-${VARIANT}
+
+ARG NODE_VERSION="none"
+RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"; fi
```

Comment:
@portellaa, do you have an idea why https://commitlint.js.org/#/ is complaining?

Reply:
Comments and correct usage of `python -m pip` is given

Reply:
@Anselmoo the problem is the missing type in the commits, for instance:
```Add devcontainer for pandas-profiling``` 
should be:
```chore: add devcontainer for pandas-profiling``` 

how to change commit messages : https://linuxize.com/post/change-git-commit-message/
the commits message convention: https://www.conventionalcommits.org/en/v1.0.0/

Reply:
Ic

Reply:
@alexbarros @portellaa, I have renamed the commit according to the https://www.conventionalcommits.org/en/v1.0.0/
thx again